src/components/keynoters/keynoters.astro

```diff
@@ -79,11 +79,9 @@ const placeholders = Math.max(0, 6 - keynoters.length);
     </g>
   </svg>
 
-  <Title className="text-primary">
-    <a href="/keynoters">Keynote Speakers</a>
-  </Title>
+  <Headline id="keynoters" title="Keynote Speakers" />
 
-  <ul class="mt-12 mb-24 flex gap-6 flex-wrap">
+  <ul class="p-0 mt-12 flex gap-6 flex-wrap">
     {
       keynoters.map((keynoter, index) => (
         <Keynoter
```

Comment:
can we add ordering here? it's differently mapped locally, than inside the preview and I'd like the order to be consistent - visually it would look better if they're turned to the proper side

![image](https://github.com/user-attachments/assets/37714ca5-664a-40bb-9545-944cd1af6fea)

Reply:
also can we have two placeholders only? 
so: 

| Brett  | Sebastián | Savannah | 
   | placeholder | placeholder |